src/Storefront/Resources/app/storefront/test/utility/history/history.util.test.js

```diff
@@ -21,7 +21,7 @@ describe('history.util tests', () => {
         HistoryUtil.push('/listing', '?page=5', { some: 'state' });
         HistoryUtil.pushParams({ 'big-image': 1, 'mode': 'detailed' }, { some: 'state' });
 
-        expect(HistoryUtil.getSearch()).toBe('?big-image=1&mode=detailed&page=5');
+        expect(HistoryUtil.getSearch()).toBe('?page=5&big-image=1&mode=detailed');
```

Comment:
Is it always reliably producing this order? Or why did you change it? 
To make it more stable, maybe the test could also use URLSearchParams?

Reply:
Yes, I ran it multiple times and could not find differences between test-runs. The difference came from moving from the "query-string" package to native URLSearchParams. Now it uses URLSearchParams in the background. URLSearchParams uses the order like it is in the object. But not exactly sure why it was sorted differently with "query-string" package. For the functionality itself does not matter.